src/style.css

```diff
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
Don't set width for BEM-block. You need to set it in another class

Reply:
added class .container